lib/lcevc/lcevc_dil.js

```diff
@@ -20,40 +20,28 @@ shaka.lcevc.Dil = class {
   /**
    * @param {HTMLVideoElement} media The video element that will be attached to
    * LCEVC Dil for input.
-   * @param {HTMLCanvasElement} canvas The canvas element that will be attached
+   * @param {Element} canvas The canvas element that will be attached
```

Comment:
Seems like this should be a more specific type like HTMLCanvasElement if that's what you expect to receive.